rllib/env/base_env.py

```diff
@@ -445,3 +452,23 @@ def reset(self):
         self.last_infos = {agent_id: {} for agent_id in self.last_obs.keys()}
         self.last_dones["__all__"] = False
         return self.last_obs
+
+
+def unwrap_gym_env(env):
```

Comment:
Could we keep this in a separate PR? Shouldn't mix up changes

Reply:
Good catch. This should not have been here.